client/index.html.template

```diff
@@ -9,5 +9,6 @@
     <-- Place your optional additional elements here -->
```

Comment:
Line 1: please use the HTML comment syntax <!-- instead of //.
Line 9, 10: similarly, use <!-- instead of <--.

Reply:
Changed